File: 프로그래머스/3/12987. 숫자 게임/숫자 게임.py
def solution(A, B):
    answer = 0
    
    A = sorted(A)
    B = sorted(B)
    A_idx = 0
    B_idx = 0
    
    while B_idx != len(B):
        if A[A_idx] < B[B_idx]:
            A_idx += 1
        B_idx +=1            
    return A_idx

File: 프로그래머스/3/12987. 숫자 게임/숫자 게임.js
function solution(A, B) {
    var answer = 0;
    A = A.sort((a,b)=>a-b)
    B = B.sort((a,b)=>a-b)
    
    let idx = 0
    for(let i = 0; i < B.length; i++){
        if(A[idx] < B[i]) {
            answer++; 
            idx++;
        }
    }
    return answer;
}